Signal/src/view controllers/OWSConversationSettingsTableViewController.m

```diff
@@ -211,6 +211,10 @@ - (void)viewDidLoad
 - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
+
+    // Since we're using a custom back button, we have to do some extra work to manage the interactivePopGestureRecognizer
+    self.navigationController.interactivePopGestureRecognizer.delegate = self;
```

Comment:
Good call.

I understand we use a custom back button as a way to space our custom header for the message view controller. But why do we use them elsewhere?

Unless you have a reason not to, I propose that in this PR I revert all VC's to use the standard back button except for the message view controller, and tweak the spacing on the custom back button so that it lines up there with the system default used elsewhere.

What do you think @charlesmchen?